src/commands/test/lib/run-tests-command.ts

```diff
@@ -77,23 +77,24 @@ export abstract class RunTestsCommand extends AppCommand {
   constructor(args: CommandArgs) {
     super(args);
 
-    if (!this.testParameters) {
-      this.testParameters = [];
-    }
-    else if (typeof this.testParameters === "string") {
-      this.testParameters = [ this.testParameters ];
+    this.testParameters = this.fixArrayParameter(this.testParameters);
+    this.include = this.fixArrayParameter(this.include);
+    
+    if (this.timeoutSec && typeof this.timeoutSec === "string") {
+      this.timeoutSec = parseInt(this.timeoutSec);
     }
+  }
 
-    if (!this.include) {
-      this.include = [];
+  protected fixArrayParameter(input: any): string[]
```

Comment:
Nitpick: You can replace this function with a single call to `_.flatten([input])`. Might make things simpler.

Reply:
When I get a mo I'll make that change. Best to keep it tidy! Thanks :)